server/src/main/java/org/opensearch/identity/internal/InternalAccessTokenManager.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.identity.internal;
```

Comment:
Lets move this package into `org.opensearch.authn.internal`, with Criags changed the only things we include in server/src/main are interfaces or noop implementations